app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -0,0 +1,44 @@
+class Chargeback
+  class ConsumptionWithRollups < Consumption
+    delegate :timestamp, :resource, :resource_id, :resource_name, :resource_type, :parent_ems,
+             :hash_features_affecting_rate, :tag_list_with_prefix, :parents_determining_rate,
+             :to => :first_metric_rollup_record
```

Comment:
for a second there I thought you got rid of `first_metric_rollup_record` and I was so excited.
Especially since I didn't' know how to get rid of it.